src/lib/components/ScheduleWidget/ScheduleWidgetMain.svelte

```diff
@@ -7,6 +7,7 @@
 	import { addMinutes, isSameDay } from 'date-fns';
 	import IconRssFeed from '../icons/IconRssFeed.svelte';
 	import IcsExport from './IcsExport.svelte';
+	import DEFAULT_EVENT_PICTURE from '$lib/assets/event-default-picture.svg';
```

Comment:
This requires for the image to be available at compile-time, for the svelte pipeline to process the asset.

It would be cumbersome to have to rebuild the project to change the image. This must be loaded dynamically.